include/d/d_particle.h

```diff
@@ -374,7 +374,7 @@ class dPa_control_c {
                               const cXyz* scale, u8 alpha, dPa_levelEcallBack* pCallBack,
                               s8 setupInfo, const GXColor* pPrmColor, const GXColor* pEnvColor,
                               const cXyz* pScale2D) {
-        return set(2, particleID, pos, angle, scale, alpha, pCallBack, setupInfo, pPrmColor, pEnvColor, pScale2D);
+        return set(1, particleID, pos, angle, scale, alpha, pCallBack, setupInfo, pPrmColor, pEnvColor, pScale2D);
```

Comment:
Checking more carefully, I think yours (2) is correct and mine (1) is wrong for setToon. I think the actual names of the inlines d_a_arrow calls to get 1 are `dComIfGp_particle_setP1` -> `dPa_control_c::setNormalP1`